Database/Corrections/sodQuestFixes.lua

```diff
@@ -3180,6 +3180,54 @@ function SeasonOfDiscovery:LoadQuests()
             [questKeys.requiredSpell] = -410095,
             [questKeys.zoneOrSort] = sortKeys.SHAMAN,
         },
+        [90206] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{1115,1116,1117,1118,208711}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 9,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Rockjaw Troggs until Rabbit Musk drops, consume it and use Tame Beast on a Rabbit critter, then speak to Toby at Amberstill Ranch to receive the rune."},
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90207] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{2009,2010,2011,2012,2013,2014,2152,209948}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Gnarlpine Furbolgs until Deer Musk drops, consume it and use Tame Beast on a Deer critter, then speak to Raelaeron next to the hunter trainer in Darnassus to receive the rune."},
+            [questKeys.requiredSpell] = -425758,
+            [questKeys.zoneOrSort] = sortKeys.HUNTER,
+        },
+        [90208] = {
+            [questKeys.name] = "Carve",
+            [questKeys.startedBy] = {{3111,3114,3113,3112,5824,5826,208184}},
+            [questKeys.finishedBy] = nil,
+            [questKeys.requiredLevel] = 1,
+            [questKeys.questLevel] = 8,
+            [questKeys.requiredRaces] = raceIDs.NONE,
+            [questKeys.requiredClasses] = classIDs.HUNTER,
+            [questKeys.objectivesText] = {"Kill Razermane Quillboars until Adder Pheromone drops, consume it and use Tame Beast on a Adder critter, then speak to Razzil in Razor Hill to receive the rune."},
```

Comment:
```suggestion
            [questKeys.objectivesText] = {"Kill Razermane Quillboars until Adder Pheromone drops, consume it and use Tame Beast on an Adder critter, then speak to Razzil in Razor Hill to receive the rune."},
```

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5608/commits/b6a6590f6bdaf0f12a1a767d8e980d573aefa508